src/System.Net.Http.Json/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <AssemblyVersion>3.2.0.0</AssemblyVersion>
+    <PackageVersion>3.2.0</PackageVersion>
```

Comment:
Should package/assembly version be `3.2.1`?

Reply:
We don't do that until we're ready to ship a new version, which is what I was alluding to in the above comment.  We shouldn't mix feature work with a merge PR.

Reply:
I see, makes sense.